client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Divider, Tab, Tabs, Collapse, useTheme} from '@material-ui/core';
+import {GroupOutlined} from '@material-ui/icons';
+import { faUserEdit } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ManualContactsForm from './ManualContactsForm/ManualContactsForm';
+import useFormStyles from 'styles/formStyles';
+import useStyles from './ContactsTabsStyles';
+
+enum contactFormTabs {
+    manualContactsForm = 0,
+    familyMembers = 1,
+}
+
+const FormComponent = ({currentTab}: {currentTab: number;}) => {
+    const classes = useStyles();
+
+    return <>
+        <Collapse classes={{container: classes.collapse}}
+                  in={currentTab === contactFormTabs.manualContactsForm}>
+            <ManualContactsForm/>
```

Comment:
add an array of objects that they have id prop which is an enum value and a form component which is the component to present

Reply:
sure